bfasst/flows.py

```diff
@@ -134,7 +134,11 @@ def flow_ic2_lse_conformal(design, flow_args, build_dir):
     design.compare_golden_files_paths.append(design.path / design.top_file)
     design.compare_golden_files_paths.extend([design.path / f for f in design.get_support_files()])
     design.golden_is_verilog = design.top_is_verilog()
-    status = conformal_cmp(design, build_dir, flow_args)
+
+    gold_netlist = str(design.netlist_path)
+    rev_netlist = str(design.reversed_netlist_path.name)
```

Comment:
Is there a reason these are cast to `str`?  And why is just the filename used for the reverse (`.name`)?  I would like to keep these as `pathlib.Path` objects when they are passed in.